packages/lesswrong/server/search/elastic/ElasticConfig.ts

```diff
@@ -366,7 +366,7 @@ const elasticSearchConfig: Record<SearchIndexCollectionName, IndexConfig> = {
   },
   Tags: {
     fields: [
-      "name^3",
+      "name^30",
```

Comment:
Flagging to @oetherington in case he has previously tuned this carefully, but I think this is fine based on the results that are coming up

Reply:
Seems fine to me - I probably spent, like, 5 seconds choosing that number